src/components/components/ebay-dialog-base/test/test.browser.js

```diff
@@ -365,10 +365,10 @@ describe('given a closed dialog with useHiddenProperty', () => {
             await component.rerender(Object.assign({}, input, { open: true }));
         });
 
-        thenItIsOpen(true);
+        useHiddenPropertyIsOpened(true);
     });
 
-    function thenItIsOpen(wasToggled) {
+    function useHiddenPropertyIsOpened(wasToggled) {
```

Comment:
Do we still have the existing test for when hidden prop is not being used?

Reply:
Yes, this one is changed since this method was copied from that method initially but not renamed. I just renamed it so it makes more sense.
https://github.com/eBay/ebayui-core/pull/1523/files#diff-75e8adbaaf3f958943e9bb789744ffc500ec020dbcba951ea7dbd946ac457f80R319-R326
Here is the original method above it